src/Servers/Kestrel/Core/src/Internal/Http/Http1Connection.cs

```diff
@@ -321,7 +321,7 @@ public void OnStartLine(HttpMethod method, HttpVersion version, Span<byte> targe
         }
 
         // Compare with Http2Stream.TryValidatePseudoHeaders
-        private void OnOriginFormTarget(HttpMethod method, HttpVersion version, Span<byte> target, Span<byte> path, Span<byte> query, Span<byte> customMethod, bool pathEncoded)
+        private void OnOriginFormTarget(bool pathEncoded, Span<byte> target, Span<byte> path, Span<byte> query)
```

Comment:
This is a good one; drops an unused Span parameter (passed via stack); and two unused registers, then moves the `bool` to front so its passed by register rather than on stack (since its after all the other params)